File: mobile/android/app/src/main/java/ru/edu/asu/minobrlabs/db/entities/Stat.java
package ru.edu.asu.minobrlabs.db.entities;

import java.io.Serializable;
import java.util.Arrays;
import java.util.Date;

public class Stat implements Serializable {
    public Long _id;

    public Integer type;
    public String vals;
    public Long date;

    public Stat() {}

    public Stat(final int type, final String vals) {
        this.type = type;
        this.vals = vals;
        this.date = new Date().getTime();
    }

    public Stat(final int type, final float[] vals) {
        this(type, Arrays.toString(vals));
    }

    @Override
    public String toString() {
        return String.format(
                "Stat : [\n  _id : %s\n  type : %s\n  vals : %s\n  date : %s\n]",
                _id,
                type,
                vals,
                new Date(date).toString()
        );
    }
}

File: desktop/windows/MinobrLabsProject/MinobrLabsProject/db/entities/Stat.cs
using System;

namespace MinobrLabsProject.db.entities
{
    public class Stat
    {
        public string vals;
        public long date;
        public long experiment;
        public string type;

        public Stat() {}

        public Stat(long date, string vals, string type, long experiment)
        {
            this.vals = vals;
            this.type = type;
            this.experiment = experiment;
        }

        public void setDate(DateTime date)
        {
            TimeSpan ts = date.ToUniversalTime() - new DateTime(1970, 1, 1);
            this.date = (long)ts.TotalMilliseconds;
        }

        public DateTime getDateTime()
        {
            DateTime dateTime = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
            dateTime = dateTime.AddSeconds(Math.Round(date / 1000d)).ToLocalTime();
            return dateTime;
        }

        public string toString() {
            return string.Format(
                    "%s : [\n  vals : %s\n  date : %s\n  experimentId : %s\n]",
                    vals,
                    getDateTime().ToString("dd.MM.yyyy HH:mm"),
                    experiment
            );
        }
    }
}
